modules/administration/pages/maintenance-windows.adoc

```diff
@@ -0,0 +1,111 @@
+[[maintenance-windows]]
+= Maintenance Windows
+
+The maintenance windows feature in {productname} allows you to schedule actions to occur during a scheduled maintenance window period.
+When you have created your maintenance window schedule, and applied it to a client, you are prevented from executing some actions outside of the specified period.
+
+
+[NOTE]
+====
+Maintenance windows operate in a different way to system locking.
+For more information about system locks, see xref:client-configuration:system-locking.adoc[].
+====
+
+
+Maintenance windows require both a calendar, and a schedule.
+The calendar defines the date and time of your maintenance window events, including recurring events, and must be in [path]``ical`` format.
+The schedule uses the events defined in the calendar to create the maintenance windows.
+You must create an [path]``ical`` file for upload, or link to an [path]``ical`` file to create the calendar, before you can create the schedule.
+
+When you have created the schedule, you can assign it to clients that are registered to the {productname} Server.
+Clients that have a maintenance schedule assigned cannot run restricted actions outside of maintenance windows.
+
+Restricted actions significantly modify the client, and could potentially cause the client to stop running.
+Some examples of restricted actions are:
+
+* Package installation
+* Client upgrade
+* Service pack migration
+* Highstate application (for Salt clients)
+
+Unrestricted actions are minor actions that are considered safe and are unlikely to cause problems on the client.
+Some examples of unrestricted actions are:
+
+* Package profile update
+* Hardware refresh
+* Subscribing to software channels
+
+
+Before you begin, you must create an [path]``ical`` file for upload, or link to an [path]``ical`` file to create the calendar.
+You can create [path]``ical`` files in your preferred calendaring tool, such as KOrganizer.
```

Comment:
Korganizer is not the best example here. Should we better name outlook/office365 and google calendar?

Reply:
I was going for open source, but sure :)

Reply:
Yes, but customers using not open source clients :-(
So we designed this to work with outlook calendars and also tried google as most people will have it.
I would still name KOrganizer, but also the other.

Reply:
Agree with MC's comment. As much as I like KOrganizer (former KDE developer here!), it's not that popular. I would rewrite naming the most popular e-mail clients:
"... such as Microsoft Outlook, Google Calendar or Mozilla Thunderbird"